GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/Garden.java

```diff
@@ -250,7 +250,9 @@ public static Garden stringToGarden(String g)
     public Species getSpeciesInfo(String speciesName)
     {
     	Species newSpecies = null;
-		
+		Date pruneDate = null;
+		Date plantDate = null;
+    	
 		//go through speciesList looking for the species with the name equal to the parameter
         for (Species s : speciesList)
         {
```

Comment:
No because it gets set to null inside of the addSpecies method. It doesnt need to get reset here, it just needs to return the species values.


________________________________
From: Charles Smith <notifications@github.com>
Sent: Wednesday, July 8, 2015 11:35 AM
To: cs1530-group1/garden_app
Cc: Alvi, Muneeb R
Subject: Re: [garden_app] Refactored getSpeciesInfo (#32)


In GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/Garden.java<https://github.com/cs1530-group1/garden_app/pull/32#discussion_r34161903>:

> @@ -250,7 +250,9 @@ public static Garden stringToGarden(String g)
>      public Species getSpeciesInfo(String speciesName)
>      {
>       Species newSpecies = null;
> -
> +             Date pruneDate = null;
> +             Date plantDate = null;
> +
>               //go through speciesList looking for the species with the name equal to the parameter
>          for (Species s : speciesList)
>          {


Shouldn't pruneDate and plantDate be set to null inside the loop so they get reset for every plant?

-
Reply to this email directly or view it on GitHub<https://github.com/cs1530-group1/garden_app/pull/32/files#r34161903>.